cwms-data-api/src/main/java/cwms/cda/data/dto/watersupply/WaterUserContractRef.java

```diff
@@ -49,10 +50,9 @@ public WaterUser getWaterUser() {
     }
 
     @Override
-    public void validate() {
-        if (this.contractName == null) {
-            throw new IllegalArgumentException("ContractName cannot be null");
-        }
-        this.waterUser.validate();
+    protected void validateInternal(CwmsDTOValidator validator) {
+        super.validateInternal(validator);
```

Comment:
use @JsonProperty(required=true) on the class variable instead of implementing validateInternal